daisy/compute/test_client.go

```diff
@@ -56,6 +56,7 @@ type TestClient struct {
 	CreateImageFn               func(project string, i *compute.Image) error
 	CreateInstanceFn            func(project, zone string, i *compute.Instance) error
 	CreateNetworkFn             func(project string, n *compute.Network) error
+	CreateSnapshotFn            func(project, zone, disk string, ss *compute.Snapshot) error
```

Comment:
Why ss instead of s?

Reply:
Usually s stands for step. I changed back to s as you suggested since there is no "step" in this place. I keep it "ss" in other places where "step" shows up.